windows/ReactTestApp/Manifest.cpp

```diff
@@ -19,6 +21,65 @@ namespace
             return std::nullopt;
         }
     }
+
+    std::any getAny(const nlohmann::json &j)
+    {
+        switch (j.type()) {
+            case value_t::null:
+                return std::nullopt;
+                break;
```

Comment:
Both `return` and `break` are jump instructions, so having both is redundant. Same below...
```suggestion
                return std::nullopt;
```